frontend_tests/node_tests/rendered_markdown.js

```diff
@@ -272,8 +277,8 @@ run_test("timestamp", ({mock_template}) => {
     // Final asserts
     assert.equal($timestamp.html(), '<i class="fa fa-clock-o"></i>\nThu, Jan 1 1970, 12:00 AM\n');
     assert.equal(
-        $timestamp.attr("title"),
-        "This time is in your timezone. Original text was 'never-been-set'.",
+        $timestamp.attr("data-tippy-content"),
+        "Everyone sees this in their own timezone.\n<br/>\nYour time zone: UTC\n",
```

Comment:
Using template is giving these `\n`s but it isn't effecting the UI. Should we be concerned about it?

Reply:
No, it's not a problem to have newline characters in the HTML.